build/rocm/run_single_gpu.py

```diff
@@ -115,7 +161,7 @@ def run_test(testmodule, gpu_tokens, continue_on_fail):
 
 
 def run_parallel(all_testmodules, p, c):
-  print(f"Running tests with parallelism=", p)
+  print(f"Running tests with parallelism = ", p)
```

Comment:
I think you might want to move `p` into the f-string?

Reply:
Thanks!